stubserver/stubserver.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask, request
+from flask_restful import Resource, Api
+
+
+app = Flask(__name__)
+api = Api(app)
+
+
+class CreateSession(Resource):
+  def post(self):
```

Comment:
What about `GET`?

Reply:
I can think of a sensible use case for GET, am happy for you to push on this branch